tests/test_cli.py

```diff
@@ -678,6 +678,37 @@ def test_run_non_universal_newline(self):
         self.assertEqual(
             (ctx.returncode, ctx.stdout, ctx.stderr), (1, expected_out, ''))
 
+    def test_run_list_files(self):
+        self.maxDiff = None
+
+        with RunContext(self) as ctx:
+            cli.run(('--list-files', self.wd))
+        self.assertEqual(ctx.returncode, 0)
+        self.assertEqual(ctx.stdout, (
+            '{dir}/c.yaml\n'
+            '{dir}/empty.yml\n'
+            '{dir}/dos.yml\n'
+            '{dir}/warn.yaml\n'
+            '{dir}/en.yaml\n'
+            '{dir}/a.yaml\n'
+            '{dir}/s/s/s/s/s/s/s/s/s/s/s/s/s/s/s/file.yaml\n'
+            '{dir}/sub/ok.yaml\n'
+            '{dir}/sub/directory.yaml/empty.yml\n').format(dir=self.wd))
+
+        config = 'ignore: |\n  *.yml\nyaml-files:\n  - \'*.*\'\n'
```

Comment:
I suggest these more readable options:
```suggestion
        config = ('ignore:\n'
                  '  *.yml\n'
                  'yaml-files:\n'
                  '- "*.*"\n')
```
```suggestion
        config = '{ignore: *.yml, yaml-files: ["*.*"]}'
```

Reply:
Great suggestion :+1: 

I had to quote `*.yml` to make it work but I like :+1:  

```suggestion
        config = '{ignore: "*.yml", yaml-files: ["*.*"]}'
```

Thank you